.github/workflows/ci.yml

```diff
@@ -47,4 +47,5 @@ jobs:
       - name: Building with bazel
         run: bazel build //...
       - name: Testing with bazel
-        run: bazel test -- //... -//integrations/... -//bindings/...
+        # TODO(gcmn) Enable all tests once they're passing
+        run: bazel test -- //... -//integrations/... -//bindings/... -//iree/hal/vulkan:dynamic_symbols_test -//iree/samples/rt:bytecode_module_api_test
```

Comment:
Tests using Vulkan will likely need SwiftShader to be configured on the test machine. See https://github.com/google/iree/blob/master/build_tools/third_party/swiftshader/build_vk_swiftshader.sh

Reply:
Cool, I'll check it out, thanks